common/src/main/java/org/apache/celeborn/common/protocol/message/StatusCode.java

```diff
@@ -74,8 +74,8 @@ public enum StatusCode {
   PUSH_DATA_CONNECTION_EXCEPTION_SLAVE(41),
   PUSH_DATA_TIMEOUT_MASTER(42),
   PUSH_DATA_TIMEOUT_SLAVE(43),
-  PUSH_DATA_MASTER_BLACKLISTED(44),
-  PUSH_DATA_SLAVE_BLACKLISTED(45),
+  PUSH_DATA_MASTER_WORKER_EXCLUDED(44),
+  PUSH_DATA_SLAVE_WORKER_EXCLUDED(45),
```

Comment:
Only used in ShuffleClientImpl, ok to change

Reply:
We are going to rename it to
```
PUSH_DATA_PRIMARY_WORKER_EXCLUDED
PUSH_DATA_REPLICA_WORKER_EXCLUDED
```

But it's OK to do in following PR, since this PR focus on blacklist